pkg/types/ccip/onramp.go

```diff
@@ -59,4 +59,14 @@ type OnRampReader interface {
 	Address() (Address, error)
 
 	GetDynamicConfig() (OnRampDynamicConfig, error)
+
+	// IsSourceChainHealthy returns true if the source chain is healthy.
+	IsSourceChainHealthy(ctx context.Context) (bool, error)
```

Comment:
Is it important to say `SourceChain` here? Or is that already implied since we are interacting with an `OnRampReader`? e.g. `onRamp.IsHealthy(ctx)`, `sourceChainOnRamp.IsHealthy(ctx)`
```suggestion
	// IsHealthy returns true if the source chain is healthy.
	IsHealthy(ctx context.Context) (bool, error)
```

Same question for `IsDestChainHealthy` vs. `IsHealthy`. e.g. `commitStore.IsHealth(ctx)`, `destChainCommitStore.IsHealth(ctx)`

Reply:
IMO it's not really important but I'm fine either way, the people that worked on this changes chose this names, I'm just migrating their changes to this repo.